objects/screen.c

```diff
@@ -94,12 +94,33 @@
  * @signal swapped
  */
 
+/** This signal is emitted when the list of physical screen area changes.
+ *
+ * Each areas in the list correspond to a **physical** screen rectangle. This
```

Comment:
TYPO: `Each areas` --> `Each area`
`correspond` --> `corresponds`

Additionally, L97, I'm not familiar with how the list is, but `list of physical screen areas` sounds correct to me.